sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -694,3 +696,35 @@ def _create_and_write_workflow(
         params_list,
         pipeline_conf)
     TektonCompiler._write_workflow(workflow=workflow, package_path=package_path)   # Tekton change
+    _validate_workflow(workflow)
+
+
+def _validate_workflow(workflow: List[Dict[Text, Any]]):  # Tekton change, signature
+  # TODO: Tekton pipeline parameter validation
```

Comment:
I took this code over from KFP unchanged as a guide but commented this code out for now since it is non-essential. I will create an issue to follow up on this.

Reply:
https://github.com/kubeflow/kfp-tekton/issues/137